Dockerfile

```diff
@@ -1,31 +1,48 @@
-FROM node:14.17 as frontend-builder
+# syntax=docker/dockerfile:1
+FROM busybox AS tmp_cache
+ARG databricks_odbc_driver_url=https://databricks.com/wp-content/uploads/2.6.10.1010-2/SimbaSparkODBC-2.6.10.1010-2-Debian-64bit.zip
+WORKDIR /tmp/
+RUN \
+  echo "Downloading SimbaSparkODBC..." \
+  && wget -O - "$databricks_odbc_driver_url" | busybox unzip -j - "*/simbaspark*.deb" -d . \
+  echo "Done!"
+
+FROM node:14.17 AS frontend-builder
 
-RUN npm install --global --force yarn@1.22.10
+RUN \
+  --mount=type=cache,target=/root/.npm \
+  npm install --global --force yarn@1.22.10
```

Comment:
As a data point, we've updated to yarn@1.22.19 a few weeks ago (241dcfacd98092be4ae2830d33b69e185f12a91a), so probably best to use that version here too.